pyproject.toml

```
@@ -97,5 +97,5 @@ extension = "html"
 profile = "django"
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core==1.5.1"]
```

https://python-poetry.org/docs/pyproject#poetry-and-pep-517 recommends the `>=` version. So perhaps

```suggestion
requires = ["poetry-core>=1.5.1"]
```

---

I'd prefer to pin this version to help avoid any unexpected side effects between different developers publishing the package with different versions of poetry